### ISSUE TITLE
Most used nn.functional ops: 1-25

### ISSUE THREAD

**Issue by Chillee** [facebookresearch/functorch#102]
- [x] softmax	10522
- [x] relu	8572
- [x] interpolate (upsample_nearest)	7277
- [x] pad (Finished everything but circular, which is in Python),	5207
- [x] log_softmax	4699
- [x] normalize (norm, clamp_min, expand_as, div)	2338
- [x] cross_entropy	2083
- [ ] grid_sample	1970
- [x] one_hot	1967
- [x] mse_loss	1920
- [x] conv2d	1593
- [x] dropout	1516
- [x] softplus	1385
- [x] sigmoid	1128
- [x] linear	1036
- [x] gelu	930
- [ ] avg_pool2d	899
- [ ] max_pool2d	876
- [ ] nll_loss	863
- [ ] embedding	737
- [x] tanh	664
- [x] leaky_relu	640
- [ ] adaptive_avg_pool2d	633
- [ ] cosine_similarity	627
- [x] unfold	609
